torntools/views/settings/settings.html

```diff
@@ -340,6 +340,8 @@ <h2>Preferences</h2>
 									</div>
 								</div>
 							</div>
+							<div class="option" id="profile-block_ally_attacks"><input type="checkbox" />Block attacks on faction mates and on ally faction members</div>
```

Comment:
We usually add the "New!" text here but I'll go over them anyway before putting out a new version so it's not that big of a thing.

Reply:
Added "New!".